cfgov/unprocessed/js/organisms/FilterableListControls.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+// Required modules.
+var atomicCheckers = require( '../modules/util/atomic-checkers' );
+var Expandable = require( '../molecules/Expandable' );
+
+/**
+ * FilterableListControls
+ * @class
+ *
+ * @classdesc Initializes a new Filterable-List-Controls organism.
+ *
+ * @param {HTMLNode} element The DOM element within which to search for the organism.
+ */
+function FilterableListControls( element ) {
+  var BASE_CLASS = 'o-filterable-list-controls';
+
+  var _dom = atomicCheckers.validateDomElement( element, BASE_CLASS, 'FilterableListControls' );
+  var _expandable = new Expandable( _dom ).init();
+
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/a66f23bd66f277f3fef60ae4e1f6de2b70da48d3